docs/release/trg-0/trg-8-03.md

```diff
@@ -0,0 +1,257 @@
+---
+title: TRG 8.03 - KIT Artefacts
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 18-Jan-2024  | Initial contribution                   |
+
+This explains each artefacts a KIT can or must follow - depending on the graduation stage (TRG 8.02)
+
+## CHANGELOG
+
+To achieve some uniformity, we prescribe a versioning scheme for KITs. Here we use a modified semantic versioning. With semantic versioning, the version is composed of three parts as follows:
+
+`MAJOR.MINOR.PATCH`
+
+An increment in Major component marks a change that makes the current version incompatible with the previous versions. A minor increment, on the other hand, indicates the addition of a new feature or component. An increment in the Patch component marks an unimportant change.
+
+For a kit we distinguish between three phases sandbox, incubating and graduated.
+
+### Sanbox Phase
+
+Has no changelog - it´s a preview for the KIT.
+
+### Incubating Phase
+
+During the first publication / release of the incubating phase, the major version must be 0. This means that the version of a KIT that is not yet graduated always has the format 0.x.y. The minor component is incremented exactly when an artifact is completed and the patch component is incremented for all other changes.
+
+As soon as a KIT moves to the second release and got published / released once it´s version is set to 1.0.0. - in general the KITS follow semantic versioning.
+
+### Graduated Phase
+
+Upon reaching the Graduation Phase, a KIT is production ready and got verified by a solution provider or data prosumer. Since the KITs are maintained and further developed by the community, changes can be made that result in an incompatibility.
+
+## ADOPTION VIEW
+
+The adoption view gives a general introduction to the KIT. Provide an understanding to an OEM, a solution provider or data provider whats the main purpose, which industry problem is this KIT solving.
+
+### Vision & Mission
+
+Mandatory for every KIT and all graduation stages
+
+The vision describes the strategic objectives of a KIT and how it aims to inspire solution providers.
+
+Deliverables:
```

Comment:
I would avoid "Deliverables". TRGs should describe quality criteria, guidelines and rules.
suggestion: 
"Answers, the 'Vision & Mission' statement of a KIT __must__ provide"